public/openapi.yml

```diff
@@ -0,0 +1,112 @@
+openapi: 3.0.1
+# We start by defining the specification version, the title, description, and version number. When a query is run in ChatGPT, it will look at the description that is defined in the info section to determine if the plugin is relevant for the user query.
```

Comment:
I might be misunderstanding something, but I was expecting `openapi.yml` to not be particularly tied to ChatGPT (even though that's what it's currently being used for), but this comment and the `description` field suggest it's more coupled to that specific service?

Reply:
I don't think it necessarily needs to mention ChatGPT, however since this openapi definition is currently only being used for ChatGPT I tried to closely mimic the example that they've released as guidance for the plug-in. As we iterate and improve we can improve on the definition and possibly explore the options of generalizing it a bit more.